### DISCUSSION TITLE
Record video problem y211ga (fw 9.0.36)

### DISCUSSION THREAD

**Discussion by Giuserver** [roleoroleo/yi-hack-Allwinner-v2#660] [Q&A]
HI,
I noticed a big mistake in recording the videos.
I have 4 cameras (different hacks, from 2.0 to 2.7) all y211ga.
3 of these have the basic fw 9.0.36.02_202105081647 and one has the basic fw 9.0.36.00_ 202011101611

the camera with fw of 2020 has no problem in recording eg: /tmp/sd/record/2023Y06M27D19H/41M00S60.mp4

the other 3, with fw 2021 have this error:
/tmp/sd/record/2023Y06M27D19H/E041M00S60.mp4

(it would seem a simple timezone problem) however by changing the timezones from App Yi Home or from Webpage (or both) or simply leaving the field Timezone from Webpage empty. They simply change the first characters before the minute of the video.
EX: /tmp/sd/record/2023Y06M27D19H/E141M00S60.mp4
OR: /tmp/sd/record/2023Y06M27D19H/E241M00S60.mp4
Etc..

The main problem is not so much the name of the file, but that at one point of the day, more particularly, while it is getting dark, the videos continue to have a duration of one minute but their **speed INCREASES**! in fact, a 60-second video will end up lasting around 45 real seconds first, up to even lasting 30 real seconds!
This can be seen by simply downloading a minute (video) from twilight onwards.. (it is no coincidence that videos are always shorter in file size!) Video frozen after 30 seconds.. (audio going x1 normal speed)

(ALL CORRECT WHEN IT'S DAY (EXCEPT THE FILE NAME) !!! Where one minute is exactly 60 seconds of video (speed x1) (I also tried disabling infrared, but same problem!)
also that it doesn't happen as soon as "night mode" is automatically activated (which by the way is impossible to deactivate)

Thank you,
Giuseppe

**Comment by roleoroleo** [comment]
I don't know how to help you.
This problem is related to the Yi firmware.
So, try to update the original firmware.

**Reply by Giuserver** [reply]
I've never updated the camera.
I just applied the hack! (by the way, this is a common problem, I'm not the first).
Also, if i reproduce videos from yi official app, playback is correct.

Thanks
Giuseppe

**Reply by roleoroleo** [reply]
I'm sure that the problem is common.
The mp4 is produced by yi software.
So, hack or not hack, the problem exists
Probably the fps decreases but the header of the mp4 doesn't change.

You could try a fw upgrade (after a full backup): 12.0.37 is available.
Maybe Yi fixed this bug.
Or you could try to downgrade using the fw extracted from the 2020 cam.

**Reply by Giuserver** [reply]
thanks for the precise answer. 
1) how can i install the firmware present in the 2020 camera inside the 2021 ones? or where i can find the 2020 fw.
2) if I'll try the 12.0.37, can i come back to 9.0.36?

Thanks!

**Reply by Giuserver** [reply]
@roleoroleo 
I should do a backup from 2020 camera and install that backup into 2021 ones?

**Reply by roleoroleo** [reply]
Yes, you should backup the fw from 2020 camera and use the unbrick procedure with the 2021 camera.
Also create a backup copy of the camera 2021.
